### ISSUE TITLE
drainHydrateMarks() doesn't seem to work when doing server-side rendering using  Webpack 4's production mode

### ISSUE THREAD

**Issue by Nimaear** [theKashey/react-imported-component#13]
I am trying to setup a modern app starter boilerplate and it all looks to work pretty nicely so far. My problem is when I am trying to run the production build (optimized for production using webpack 4)

To reproduce:
1. Clone https://github.com/Nimaear/react-redux-starter
2. `npm install`
3. `npm start`
(You'll see that everything seems to work properly: `window.___REACT_DEFERRED_COMPONENT_MARKS` is populated and hydrating works good.)

4. Now build the app for production using `npm run build` and `npm run prod` (you'll notice that `window.___REACT_DEFERRED_COMPONENT_MARKS` is empty now and _loading_ occurs.)

Am I missing something here?

**Comment by theKashey** [comment]
It is just super smart. As long imported component get `mark` from function code (it .toString) - it __don't use__ it in the "real code".
Webpack sees that, and cast mark to "0"
```js
// normal code
return x(0, 0, n.e(2).then(n.bind(null, "./src/views/About/About.js")))
// patched code (side-effected)
return x(0, "snc9gq", n.e(2).then(n.bind(null, "./src/views/About/About.js")))
```

**Comment by theKashey** [comment]
Thank you so much for this issue. Fixed in v4.3.2
